app/controllers/employees_controller.rb

```diff
@@ -46,4 +47,10 @@ def destroy
   def employee_params
     params.require(:employee).permit(:name, :email, :password, :role, :company_id, :department_id)
   end
+
+  def generate_password
+    o = [('a'..'z'), ('A'..'Z')].map(&:to_a).flatten
+    string = (0...10).map { o[rand(o.length)] }.join
```

Comment:
no need to assign into string variable as the last executed statement is returned